src/table/table.component.ts

```diff
@@ -375,6 +375,8 @@ import { I18n } from "./../i18n/i18n.module";
 						[ibmDataGridFocus]="isDataGrid"
 						[(columnIndex)]="columnIndex"
 						[attr.colspan]="row.length + 2"
+						(mouseenter)="addHoverClass($event)"
```

Comment:
could we just use the parent `tr` for this?

Some extra work, but this would be another good opportunity to add a directive 😏 (totally optional)